etc/submit_py3.sh

```diff
@@ -142,10 +144,10 @@ echo "======== WMAgent COMP Python bootstrap starting at $(TZ=GMT date) ========
 
 # First, decide which COMP ScramArch to use based on the required OS and Architecture
 THIS_ARCH=`uname -m`  # if it's PowerPC, it returns `ppc64le`
-# if this job can run at any OS, then use rhel7 as default
+# if this job can run at any OS, then try to run it on the OS estimated at runtime
```

Comment:
How about rephrasing it to "discovered" (instead of "estimated")? As this is what is actually happening.

Reply:
done